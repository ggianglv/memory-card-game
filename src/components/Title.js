import React from 'react';

const Title = () => {
  return (
    <h3 className="title">
      Memory Card game
    </h3>
  );
};

export default Title;
