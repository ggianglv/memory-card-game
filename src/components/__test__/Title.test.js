import React from "react";
import "@testing-library/jest-dom/extend-expect";
import Title from "../Title";
import renderer from "react-test-renderer";

test("Title to match snapshot", () => {
  const tree = renderer.create(<Title />).toJSON();

  expect(tree).toMatchSnapshot();
});
