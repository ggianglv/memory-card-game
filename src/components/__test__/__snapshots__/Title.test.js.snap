// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title to match snapshot 1`] = `
<h3
  className="title"
>
  Memory Card game
</h3>
`;
